src/status_im/data_store/switcher_cards.cljs

```diff
@@ -0,0 +1,39 @@
+(ns status-im.data-store.switcher-cards
+  (:require [utils.re-frame :as rf]
+            [taoensso.timbre :as log]
+            [clojure.set :as clojure.set]))
+
+(defn <-rpc
+  [switcher-card]
+  (clojure.set/rename-keys switcher-card
```

Comment:
I believe we're following the Clojure Style Guide for certain namespace aliases. See the table here https://guide.clojure.style/#use-idiomatic-namespace-aliases

`clojure.set` should be aliased as `set`